src/client/Ruby/ms-rest-azure/lib/ms_rest_azure/polling_state.rb

```diff
@@ -98,7 +98,14 @@ def get_operation_error
     
     def get_request(options = {})
       link = @azure_async_operation_header_link || @location_header_link
-      options[:connection] = create_connection(options[:base_uri])
+      faraday_options = {
+          middlewares: request.middlewares,
+          headers: request.headers,
+          log: request.log
```

Comment:
Here's some info:
- http://stackoverflow.com/questions/14319347/variables-in-ruby-on-rails 
- http://stackoverflow.com/questions/9809843/understanding-ruby-class-attributes-using-accessor-macros-and-self
I can update this to use @, since we're inside the class it's probably clearer to use it, though I've seen other cases already in the code that don't use it.